gulpfile.js

```diff
@@ -167,6 +169,38 @@ gulp.task('serve', ['build'], function() {
 });
 
 /**
+ * Build the application, move it with scp.
+ * Takes SSH host, username, password and folder to move the app as parameters
+ *
+ * For example: gulp scp --host=localhost --user=moi --pw=kaikki --dest=/home/
+ * Remember to set API and IO env variables to point to right machine!
+ *
+ * IO_URL=http://192.168.142.21:9001 API_URL=http://192.168.142.21:9002/api  gulp scp
+ */
+gulp.task('scp', ['build'], function() {
+	if (!process.env.IO_URL)   console.warn("WARNING: IO_URL environment variable not set!");
+	if (!process.env.API_URL)  console.warn("WARNING: API_URL environment variable not set!");
+
+	var host     = args['host'] ? args['host'] : "192.168.142.12";
```

Comment:
Maybe don't hardcode URL's and passwords. I'd also prefer if we left the SSH options such as host and user etc. in environment variables, such as:
```
var host = process.env.SCP_HOST || 'localhost';
```